jest.config.js

```diff
@@ -1,7 +1,7 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```

Comment:
is this necessary?

Reply:
it is not required, it is from official documentation and give you some completion.
<img width="630" alt="image" src="https://user-images.githubusercontent.com/8377185/164111140-1ce16285-b466-452f-b0a3-cb7499c6700f.png">

Do you want to remove it?

Reply:
Oh nice 👍